scripts/west_commands/tests/test_imports.py

```diff
@@ -43,6 +43,7 @@ def test_runner_imports():
                     'spi_burn',
                     'stm32cubeprogrammer',
                     'stm32flash',
+                    'teensy',
```

Comment:
@nashif it seems that the addition of `teensy` to this file and `__init__.py` is flagging the pylint duplicate code check, causing CI to fail. Should we just ignore the CI failure in this case? Or is there a fix we could apply to the pylint check?

Reply:
Splitting it up into 2 commits seems to solve it.

Reply:
Perfect, wasn't sure if CI was running per commit or looking at the entire PR diff